AppCenterAuth/AppCenterAuth/Internals/Models/MSAuthority.m

```diff
@@ -2,32 +2,27 @@
 // Licensed under the MIT License.
 
 #import "MSAuthority.h"
+#import "MSB2CAuthority.h"
```

Comment:
You should sort `import`s. Please run `./clang-format-changed-files.sh` in your repo root to format all files that you touched.